src/accelerate/accelerator.py

```diff
@@ -273,6 +273,10 @@ def __init__(
         if self.rng_types is None:
             self.rng_types = ["torch"] if is_torch_version("<=", "1.5.1") else ["generator"]
 
+    @property
+    def use_distributed(self):
+        return self.distributed_type != DistributedType.NO
```

Comment:
Maybe also check for the number of processes to be `> 1` here, since you are replacing all those tests?